app/components/Welcome/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import ContentPage from 'components/ContentPage'
+import gnosisOwl from 'assets/gnosis_owl.svg'
+import styles from './index.css'
+
+const Welcome = () => (
+  <ContentPage>
```

Comment:
Why you do not use ```<Page>``` here?

Reply:
ContentPage do not display the header

Reply:
in that case I recommend to do something:
```
<Page withoutHeader ...>
    ....
</Page>
```
And then, inside Page:
```
const Page = ({withoutHeader = false}) => (
  <div ...>
      { !withoutHeader && <Header ..../> }
      ....
  </div>
```
All pages routes/pages should follow the same layout. In that way, if you apply a change to Page component you know all routes will be updated. 

Keep the good job.